### DISCUSSION TITLE
Error in hook walkComponentParents

### DISCUSSION THREAD

**Discussion by acetilholin** [quasarframework/quasar#11518] [General - Components / Directives / etc] [answered]
I'm using  @quasar/cli - 1.2.2
Every time I refresh any page in my web app, I'm getting error below:

`An error occurred in hook walkComponentParents`
`TypeError: Cannot read properties of null (reading 'instanceMap')
    at ComponentWalker.mark (backend.js:5150)
    at ComponentWalker.captureId (backend.js:5071)
    at ComponentWalker.getComponentParents (backend.js:4985)
    at backend.js:5310
    at DevtoolsHookable.callHandlers (backend.js:723)
    at DevtoolsApi.callHook (backend.js:317)
    at DevtoolsApi.walkComponentParents (backend.js:374)
    at Object.<anonymous> (backend.js:1625)`

Is this a Vue issue or Quasar's? Any solutions?

**Comment by MilosPaunovic** [comment]
Are you using any external libraries in your project? I've stumbled onto [this](https://bestofvue.com/repo/NightCatSama-vue-slider-component-vuejs-slider) thread with one of the replies stating:

```bash
After I added this library to my project and this error appeared, If I remove the library the error dissapear:
```

regarding `vue-slider-component`

**Reply by acetilholin** [reply]
No, zero external libraries.

**Comment by MilosPaunovic** [comment, marked as answer]
Have you tried the default solution, to remove `node_modules` folder and `package-lock.json` and run a fresh `npm i`?

**Comment by pdanpdan** [comment]
I suppose it's an error from vue dev tools.
They still have a lot of problems.

**Comment by mnguyen284** [comment]
I have created a new project using Quasar CLI 1.2.2. Try quasar dev, everything is fine.
Just by adding 'Notify' to plugins in quasar.conf.js the same error shows up.

**Reply by mnguyen284** [reply]
The error is gone when I updated Vue devtools Chrome extension to version 6.0.8.

**Reply by yaros-eu** [reply]
@mnguyen284, I'm getting the same issue with version 6.0.12. Do you still have it working?
